pillar/heroku/mitopen.sls

```diff
@@ -128,12 +127,12 @@ heroku:
     OCW_NEXT_AWS_STORAGE_BUCKET_NAME: {{ env_data.OCW_NEXT_AWS_STORAGE_BUCKET_NAME }}
     OCW_NEXT_BASE_URL: {{ env_data.OCW_NEXT_BASE_URL }}
     OCW_NEXT_LIVE_BUCKET: {{ env_data.OCW_NEXT_LIVE_BUCKET }}
-    OCW_NEXT_SEARCH_WEBHOOK_KEY: __vault__::secret-{{ business_unit }}/global/update-search-data-webhook-key>data>value
+    OCW_NEXT_SEARCH_WEBHOOK_KEY: __vault__::secret-mitopen/secrets>data>data>global>update-search-data-webhook-key
```

Comment:
Again, for values shared across projects/apps we should just use the `secret-operations` mount